databases/mongo/models/familiarities.js

```diff
@@ -63,25 +62,27 @@ module.exports = {
   },
 
 
-  getOrderedCardIds(userId, callback) {
+  getOrderedCardIds(userId, cutoff, callback) {
```

Comment:
This is definitely cleaner than requiring a whole file while still maintaining modularity